### PR TITLE
Including cme standard copyright

### PR DESCRIPTION
**Pull request by diegomillan**
### Description

This PR is a theme customization in order to include a standard copyright footer at the bottom of LMS pages. 
### Reviewers

If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [ ] Code review: @stvstnfrd
- [ ] Code review: @felipemontoya

FYI: Tag anyone who might be interested in this PR here

@juancamilom
### Post-review
- [ ] Rebase and squash commits

### CODE REVIEW

**Comment by felipemontoya** [comment]
This is ready for the @stvstnfrd review. :+1: from me.

**Comment by stvstnfrd** [comment]
@diegomillan @felipemontoya 

This looks like you're just using the standard licenscing feature,
but forking the template? A couple of thoughts.
- When I originally suggested, "do this in a theme", it's because I
  thought Mark wanted the footer on _every_ _single_ page, not just
  courseware pages.
- The former wasn't possible with the licensing feature, hence my
  suggestion.
- This looks like Mark just wants the latter (good, it's more
  maintainable). In that case, theming is overkill, let's just enable
  licensing and call it a day.

**Comment by diegomillan** [comment]
Hi @stvstnfrd we have updated this PR according to Mark's feedback. He would like to have the copyright on every LMS page and not just inside the course. Note we made changes only to the CME theme.

We are ready for your feedback here, thanks.

File: templates/theme-footer.html [hunk 1]
```diff
@@ -3,10 +3,14 @@
   from django.core.urlresolvers import reverse
   from django.utils.translation import ugettext as _
   from edxmako.shortcuts import marketing_link
+  from django.utils.timezone import now
 %>
 <div class="wrapper-footer">
   <footer id="footer-openedx">
     <div class="colophon">
+      <div class="copy-colophon">
+        © ${now().strftime('%Y')} <span class="license-text">${_("Stanford Medicine")}</span>
```
Comment (stvstnfrd) [inline templates/theme-footer.html]: Lets use the `&copy;` htmlentity here, instead of directly embedding the symbol.

Comment (stvstnfrd) [reply templates/theme-footer.html]: `&copy; 20XX` should be wrapped in a `SPAN` tag, as text shouldn't appear immediately within a blocklevel element

Comment (stvstnfrd) [reply templates/theme-footer.html]: For consistency in Python/JS, we use single quotes for non-interpolated strings; `$(_('Stanford Medicine'))`.

**Review by stvstnfrd — CHANGES_REQUESTED** [review]
Some small comments, but otherwise looking good.

> He would like to have the copyright on every LMS page and not just inside the course.

Glad we confirmed :)

**Comment by diegomillan** [comment]
@stvstnfrd I applied the requested changes.

File: templates/theme-footer.html (hunk 1 above)
Comment (diegomillan) [reply templates/theme-footer.html]: @stvstnfrd I applied the requested changes.

**Review by stvstnfrd — APPROVED** [review]

**Pull request merged**